<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>segmentationClass</class>
 <widget class="QMainWindow" name="segmentationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1872</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>segmentation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/segmentation/art_design/background.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>521</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/segmentation/art_design/2.png);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="OpenFig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom.png);
color: rgb(0, 0, 0);
font: 75 13pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>检查路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="index_check">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/1.png);
font: 28pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>进度</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>110</y>
      <width>541</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/segmentation/art_design/2.png);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>520</width>
       <height>490</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="page_up">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>630</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom.png);
color: rgb(140, 0, 255);
font: 16pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>上一张</string>
    </property>
   </widget>
   <widget class="QPushButton" name="page_down">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>640</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom.png);
color: rgb(140, 0, 255);
font: 16pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>下一张</string>
    </property>
   </widget>
   <widget class="QLabel" name="index_check_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>680</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/1.png);
font: 18pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>医生判读结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="doctor_answer">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>680</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/1.png);
font: 18pt &quot;Corbel&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="big">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>460</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom.png);
color: rgb(140, 0, 255);
font: 16pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>放大</string>
    </property>
   </widget>
   <widget class="QPushButton" name="small">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>530</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom.png);
color: rgb(140, 0, 255);
font: 16pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>缩小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom.png);
color: rgb(0, 0, 0);
font: 75 13pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>结果路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>680</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
border-image: url(:/segmentation/art_design/bottom3.png);
font: 12pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>       需要再次检查</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>531</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/segmentation/art_design/2.png);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="index">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/1.png);
font: 28pt &quot;AcadEref&quot;;
color: rgb(57, 173, 84);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="save_result_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>1781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom4.png);
font: 25 11pt &quot;Bahnschrift Light Condensed&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom4.jpg);
color: rgb(0, 0, 100);
font: 20pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="index_check_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom4.png);
color: rgb(0, 0, 100);
font: 10pt &quot;AcadEref&quot;;</string>
    </property>
    <property name="text">
     <string>直接跳转到</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/segmentation/art_design/bottom4.png);
font: 25 11pt &quot;Bahnschrift Light Condensed&quot;;
color: rgb(0, 0, 100);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1872</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="segmentation.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>segmentationClass</receiver>
   <slot>go_to()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>731</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go_to()</slot>
  <slot>on_gamma_clicked()</slot>
 </slots>
</ui>
